e2e/src/pages/multimedia.po.ts

```diff
@@ -0,0 +1,17 @@
+import {AppPage} from './app.po';
+import {by} from 'protractor';
+
+export class MultimediaPage extends AppPage {
+
+  async clickMultimedia() {
+    await this.clickElement(by.id('multimedia-button'));
+  }
+
+  async clickPlayButton() {
+    await this.clickElement(by.css('#mainContainer > div > mv-multimedia-player'));
```

Comment:
pause and play are exactly the same, so why create two functions?
also, we can just use this selector, can't we "mv-multimedia-player"?

Reply:
fixed.
Yes we can use css selector ref as "mv-multimedia-player"